### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by aixtools** [OpenPrinting/cups#374] [General] [answered]
- Thanks for being here.
- Google only seems to find the apple cups library, and when not on macos, not on linux, not on windows - life gets interesting and challenging.
- I made a fork - and I hope: a) my PR's will be useful; b) acceptable (as they will be supporting AIX).
- Why cups for AIX (as it has both SysV and Berkley support built-in)? - Because the OpenJDK sources require support of the CUPS-API.

**Comment by aixtools** [comment, marked as answer]
Not creating an issue - I'll build autoconf-2.71, but these tools often do not work right away on AIX - there are few people who test them. So I hope there is something really really important to need an update. 2.69 has worked so well for so long.

p.s. - this is the first time I have used github 'discussion' - wasn't expecting it to be a Q&A forum.

------------------------------------------------------------------------------------------------------------------------------------

Edit by @zdohnal to share the answer from @michaelrsweet reply:

```
WRT autoconf 2.71, IIRC it is needed for the --sharedstatedir option which some build systems are specifying now when running a configure script.
```

**Reply by michaelrsweet** [reply]
@aixtools Yes, this is Github's answer for a public mailing list/forum solution...

WRT autoconf 2.71, IIRC it is needed for the `--sharedstatedir` option which some build systems are specifying now when running a configure script.

**Reply by aixtools** [reply]
ok. I'll try hacking it with 2.69 for now.
